src/app/types/roles/CustomerRolesCollection.ts

```diff
@@ -0,0 +1,9 @@
+import { Collection, ObjectId } from 'mongodb';
+
+export interface CustomerRolesCollection extends Collection {
+	_id: ObjectId,
+    customerId: string,
+    roles: {
+        rolesMap: object,
```

Comment:
I think you should define the type here explicitly, as a developer I want to know exactly how the rolesMap is structured. You have it as a Map of `snowflake` to `string`, so define that explicitly. Object in typescript is basically `any`, so you can use `Record<Snowflake, string>`. 

You might also consider storing as an array, because the role names are not fixed.